### PR TITLE
List splittable types

### PR DESCRIPTION
**Pull request by thehobbit85**

### CODE REVIEW

File: src/edge-core-index.js [hunk 1]
```diff
@@ -397,6 +397,7 @@ export type EdgeAccount = {
   getFirstWalletInfo(type: string): ?EdgeWalletInfo,
   getWalletInfo(id: string): EdgeWalletInfo,
   listWalletIds(): Array<string>,
+  listSplittableWalletTypes(walletId: string): Array<string>,
```
Comment (swansontec) [inline src/edge-core-index.js]: We still need to add this function to the `EdgeCurrencyPlugin` type.

File: src/modules/account/accountState.js [hunk 2]
```diff
@@ -483,6 +483,23 @@ class AccountState {
...
 
+  listSplittableWalletTypes (walletId) {
+    const walletInfo = this.allKeys.find(
+      walletInfo => walletInfo.id === walletId
+    )
+    if (!walletInfo) throw new Error(`Invalid wallet id ${walletInfo.id}`)
+    const plugin = getCurrencyPlugin(
+      this.ai.props.output.currency.plugins,
+      walletInfo.type
+    )
```
Comment (swansontec) [inline src/modules/account/accountState.js]: There is no error handling for if `plugin` is `undefined`.

File: src/edge-core-index.js (hunk 1 above)
Comment (thehobbit85) [reply src/edge-core-index.js]: added

File: src/modules/account/accountState.js (hunk 2 above)
Comment (thehobbit85) [reply src/modules/account/accountState.js]: added

**Pull request merged**